eradiate/solvers/core/_runner.py

```diff
@@ -40,7 +40,10 @@ def runner(kernel_dict):
     # Run computation
     from eradiate.kernel.core.xml import load_dict
 
-    kernel_scene = load_dict(kernel_dict)
+    try:
+        kernel_scene = load_dict(kernel_dict.data)
+    except AttributeError:
+        kernel_scene = load_dict(kernel_dict)
```

Comment:
Is this for backward compatibility? I think this should be removed in the final version then, right?

Reply:
I did that to tolerate regular dicts, but it's indeed safer to just require a `KernelDict`. I'll change that.